src/erlang_ls_db.erl

```diff
@@ -55,6 +55,18 @@ store(Table, Key, Value) ->
   true = ets:insert(Table, {Key, Value}),
   ok.
 
+-spec update(table(), key(), any(), any()) -> ok.
+update(Table, Key, Old, New) ->
+  Replace = [{{Key, Old}, [], [{const, {Key, New}}]}],
+  case ets:select_replace(Table, Replace) of
```

Comment:
This is the new function that has been made available in a "recent" OTP release.

Reply:
That is OTP 20.0